src/memory/RomMatraInk.cc

```diff
@@ -5,17 +5,9 @@
 
 namespace openmsx {
 
-static constexpr auto sectorInfo = [] {
-	// 2 * 64kB, fully writeable
-	using Info = AmdFlash::SectorInfo;
-	std::array<Info, 2> result = {};
-	ranges::fill(result, Info{64 * 1024, false});
-	return result;
-}();
-
 RomMatraInk::RomMatraInk(const DeviceConfig& config, Rom&& rom_)
         : MSXRom(config, std::move(rom_))
-        , flash(rom, AmdFlashChip::AM29F040, sectorInfo,
+        , flash(rom, AmdFlashChip::AM29F040, {},
```

Comment:
This does introduce a functional difference. But possibly/probably this patch is an improvement (closer to real hardware)??

Before we only had 2 * 64kB. Now (via the AM29F040 chip) we have 8 * 64kB.

Reply:
Nice spot, I realised this when I implemented it but then forgot about it. I just double-checked, and functionally there’s no difference since it only peeks, reads and writes with 16-bit addresses.